src/include/win_remote_shell.h

```diff
@@ -50,10 +50,10 @@
 extern void std_output(char *buf);
 extern void std_error(char *buf);
 extern void disconnect_close_pipe(HANDLE hPipe);
-extern int create_std_pipes(STARTUPINFO* psi, char *pipename_append, int is_interactive);
-extern int do_ConnectNamedPipe(HANDLE hPipe, LPOVERLAPPED pOverlapped);
+extern DWORD create_std_pipes(STARTUPINFO* psi, char *pipename_append, int is_interactive);
```

Comment:
The return type of GetLastError() is DWORD. At most places we are returning the error code we get from GetLastError(). Since we are refactoring the code, it's better to use the correct data type.